File: example/src/main/java/ru/virgil/example/box/BoxDto.java
package ru.virgil.example.box;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import ru.virgil.example.system.IdentifiedDto;

@NoArgsConstructor
@AllArgsConstructor
@EqualsAndHashCode(callSuper = true)
@Data
public class BoxDto extends IdentifiedDto {

    private BoxType type;
    private String description;
    private int price;
    private float weight;

}

File: example/src/main/java/ru/virgil/example/box/BoxDto.kt
package ru.virgil.example.box

import ru.virgil.example.system.dto.IdentifiedDto
import java.time.LocalDateTime
import java.util.*

data class BoxDto(
    override var uuid: UUID? = null,
    override var createdAt: LocalDateTime? = null,
    override var updatedAt: LocalDateTime? = null,
    var type: BoxType? = BoxType.USUAL,
    var description: String?,
    var price: Int? = 0,
    var weight: Float? = 0f,
) : IdentifiedDto
